.circleci/prepare.sh

```diff
@@ -0,0 +1,8 @@
+$PYTHON --version
+$PYTHON -m pip --version
+$PYTHON -m pip install -q --user --ignore-installed --upgrade virtualenv
+$PYTHON -m virtualenv -p $PYTHON venv
+venv/bin/pip install -U pip
```

Comment:
Why would I `./` when I've already got a path specified?  `./` is just a way to get a path when you don't otherwise need one.  At least as far as I know...

Reply:
I guess I picked up the habit somewhere and I’m not sure why.  It seems less ambiguous, because while slashes do disable $PATH-searching, it doesn’t *inherently* do that. Maybe it works differently in `zsh`?